src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -71,8 +72,13 @@ export const ArtworkSidebarCommercialButtons: React.FC<
       extractNodes(me.partnerOffersConnection)[0]) ||
     null
 
+  const isLotClosed = lotIsClosed(artwork.sale, artwork.saleArtwork)
+
   // Fall back to a definitely past value because the timer hook doesn't like nulls
-  const partnerOfferTimer = useTimer(partnerOffer?.endAt || THE_PAST)
+  const partnerOfferTimer = useTimer({
+    endDate: partnerOffer?.endAt || THE_PAST,
+    enabled: !isLotClosed,
```

Comment:
Shouldn't this be based on the offer rather than lot status?

Reply:
Just set up so that if the lot is closed, we don't poll. I'm not too sure beyond that.

Reply:
It seems to me that we can just have use timer disable itself once it reaches the end date. And then we wouldn't need to explicitly enable or disable in these cases.